packages/amplify-cli-core/src/help/commands-info.ts

```diff
@@ -0,0 +1,1257 @@
+export const commandsInfo = [
+  {
+    "command": "init",
+    "commandDescription": "Initialize a new Amplify project",
+    "commandUsage": "amplify init [flags]",
+    "learnMoreLink": "https://docs.amplify.aws/cli/commands/init",
```

Comment:
Docs links should be constructed instead of hardcoding.

```ts
const amplifyCliDocsUrl = `https://docs.amplify.aws/cli`;

const getDocsLinkForCommand = (commandName: string) => 
  `${amplifyCliDocsUrl}/commands/${commandName}`;

const getDocsLinkForHeadlessUsage = () => 
  `${amplifyCliDocsUrl}/usage/headless`;
```

```ts
        learnMoreLink: getDocsLinkForCommand('env'),
```